src/main/java/org/craftercms/studio/controller/rest/v2/ContentController.java

```diff
@@ -264,8 +265,27 @@ public ResponseBody itemUnlockByPath(@RequestBody @Valid UnlockItemByPathRequest
         return responseBody;
     }
 
+    @Valid
+    @GetMapping(PATH_PARAM_SITE)
+    public ResponseEntity<Resource> getContent(@ValidSiteId @PathVariable(value = REQUEST_PARAM_SITE) String siteId,
+                                          @ValidExistingContentPath @RequestParam(value = REQUEST_PARAM_PATH) String path,
+                                          @EsapiValidatedParam(type = ALPHANUMERIC) @RequestParam(value = REQUEST_PARAM_COMMIT_ID, required = false) String commitId)
+            throws ServiceLayerException, UserNotFoundException {
+        if (commitId == null) {
+            commitId = HEAD;
+        }
+        Resource resource = contentService.getContentByCommitId(siteId, path, commitId).orElseThrow();
```

Comment:
Let's create a simple private method to prevent code duplication. `getContentByCommitId` and `getContent` methods are identical

Reply:
I updated this. Thank you!